class/defaults.yml

```diff
@@ -25,6 +25,7 @@ parameters:
       enabled: false
       annotations: {}
       secretName: ${keycloak:fqdn}
+      controllerNamespace: ingress-nginx
```

Comment:
We used to configure this via component-networkpolicy and labels.

```
  networkpolicy:
    allowNamespaceLabels:
      - network.openshift.io/policy-group: ingress
      - network.openshift.io/policy-group: hostNetwork
```

But I'm open minded for this solution too.

Reply:
The problem lies not with OpenShift, but with Rancher resp. Calico network plugin.
I'm fairly confident that openshift's network plugin resp. the already-provisioned network policies work out of the box.
However, in our Syn hierarchy we have not enabled the networkpolicy component for Rancher by default (yet?).

In other words, for Openshift a custom network policy might not even be needed, since the allow-from-ingress policy already allows ingress to connect, and cross-namespace traffic is already prohibited by design. Would you agree that on OCP the custom Keycloak netpol can be left disabled since the default policies already allow the Ingress resp. deny others?

Reply:
Yes, makes sense. OCP netpol management is kind of offloaded to component-networkpolicy and as there is nothing else in the same namespace adding additional netpols is kind of overkill.